library/data/registry-item-attributes/src/main/java/org/quiltmc/qsl/registry/attribute/impl/QuiltRegistryInternals.java

```diff
@@ -0,0 +1,9 @@
+package org.quiltmc.qsl.registry.attribute.impl;
+
+public interface QuiltRegistryInternals {
+	BuiltinRegistryItemAttributeHolder<?> qsl$getBuiltinItemAttributeHolder();
+	void qsl$setBuiltinItemAttributeHolder(BuiltinRegistryItemAttributeHolder<?> holder);
+
+	BuiltinRegistryItemAttributeHolder<?> qsl$getItemAttributeHolder();
+	void qsl$setItemAttributeHolder(BuiltinRegistryItemAttributeHolder<?> holder);
```

Comment:
This should be called `qsl$getDataItemAttributeHolder`, since it's unclear that it's supposed to hold the data attributes and not the combined attribute holder